From: Jan Engelhardt <jengelh@inai.de>
Date: 2016-04-02 15:38:45+0000

lrslib broke the ABI between e.g. v6.1 and v6.2 (added
lrsdat::countonly) without updating the SONAME. I am not trusting the
authors anymore.

---
 makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: lrslib-070/makefile
===================================================================
--- lrslib-070.orig/makefile
+++ lrslib-070/makefile
@@ -191,7 +191,7 @@ fourier:	fourier.c lrslib.h lrslib.c lrs
 # From here on the author is David Bremner <bremner@unb.ca> to whom you should turn for help             
 #
 # Shared library variables
-SONAME ?=liblrs.so.0
+SONAME ?=liblrs-${PACKAGE_VERSION}.so.0
 SOMINOR ?=.0.0
 SHLIB ?=$(SONAME)$(SOMINOR)
 SHLINK ?=liblrs.so
